### DISCUSSION TITLE
Filecoin币价冲上200美金适合参与挖矿吗？

### DISCUSSION THREAD

**Discussion by DaYu-IPFS** [filecoin-project/community-china#55] [社区公告]
2021年3月30日国内上市的大型游戏商第九城宣布，再次加码签署200万美金的Filecoin服务器采购及运营托管协议，再次购买Filecoin矿机服务器集群及技术服务。

**今年2月**，九城已经和一家服务商签定了价值1000万美元的Filecoin矿机群组采购及技术服务框架协议，进行Filecoin采矿。此次的新协议是九城和另一个服务商所签署，在Filecoin业务上再次加码，期望在Filecoin呈现高速发展趋势的初期，继续增加算力及Filecoin的收入。

**在3月初**，新元科技投入5.8亿参与Filecoin挖矿，这些报道的都是一些上市公司，还有很大没有上市的公司在默默布局Filecoin挖矿，还有很多我们不知道。

**这么多的企业动不动就是上千万、上亿的投入Filecoin挖矿，是挖矿哦：他们的布局都是面对的未来，这样说明挖矿的收益在未来是大于屯币的回报。**

这就是一个未来的趋势风向，那么我们在纠结是否屯币还是挖矿时，那就分散投资，一部分选择屯币，一部分选择跟上市企业的做法走Filecoin挖矿布局。屯币的币价波动会比较大，建议保守的还是老实挖矿吧！但要选择好比较靠谱的矿商，否则一样是一场空。

当前FIL的实际流通量只有三千多万枚，这些企业上亿的投入Filecoin挖矿，所需的FIL质押币，GAS费从哪里来，还不是从市场上购买，流通量就这么点，价格能不能上涨吗。

**Comment by DaYu-IPFS** [comment]
想要参与挖矿的投资者，可以联系我 15228181858（微信同号）

**Reply by lee620422** [reply]
FILECOIN联合挖矿，算力销售，节点可查，香港IDC机房，安全有保障。微信滴滴：HBT_CNN

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308

**Comment by zhaobo443** [comment]
V:zhaobo9308

**Comment by RichByDMC** [comment]
挖FIL的伙伴们看过来，一个火爆全网的全新存储项目自带公链钱包NFT等完整生态DMC，唯一能够超过Fil的项目诞生了。主要还能双挖FIL和DMC~~有想法的可以V我：dmctorich
